src/common/request.js

```diff
@@ -54,6 +55,7 @@ export default class Request {
 
         request.end()
       } else {
+        const req = this._request
```

Comment:
I'm not super familiar with the browser request component of this, does it not also support `https`?